### PR TITLE
Alman2

### PR DESCRIPTION
**Pull request by Almansherov**
Take a look

### CODE REVIEW

**Comment by Almansherov** [comment]
This is bullshit request. Seems like I was working on my forked repository, my own branch. What a failure

File: DlgCreatePeer.cpp [hunk 1]
```diff
@@ -101,31 +101,31 @@ void DlgCreatePeer::create_button_pressed(){
...
+        ui->lbl_err_name->setText(tr("Name cannot be empty"));
         ui->lbl_err_name->setStyleSheet("QLabel {color : red}");
         ui->lbl_err_name->show();
         ui->btn_create->setEnabled(true);
         errors_exist = true;
     }
     else
     if(ui->le_name->text().contains(m_invalid_chars) || ui->le_name->text().contains("_")){
-            ui->lbl_err_name->setText(tr("You can use only alphabetic characters and digits"));
+            ui->lbl_err_name->setText(tr("You can use only letters and numbers"));
```
Comment (Erkin97) [inline DlgCreatePeer.cpp]: I think digits is more correct

File: commons/src/Commons.cpp [hunk 2]
```diff
@@ -37,49 +37,49 @@ CCommons::NetworkErrorToString(int err_code) {
...
+    case QNetworkReply::ProtocolUnknownError: return QObject::tr("Error: Protocol Unknown ");
+    case QNetworkReply::ProtocolInvalidOperationError: return QObject::tr("Error: Protocol Invalid Operation Error");
     case QNetworkReply::ProtocolFailure: return QObject::tr("Protocol Failure");
 
       // Server side errors (401-499)
-    case QNetworkReply::InternalServerError: return QObject::tr("Internal Server Error");
-    case QNetworkReply::OperationNotImplementedError: return QObject::tr("Operation not implemented error");
-    case QNetworkReply::ServiceUnavailableError: return QObject::tr("Service unavailable error");
-    case QNetworkReply::UnknownServerError: return QObject::tr("Unknown server error");
+    case QNetworkReply::InternalServerError: return QObject::tr("You cannot unregister peer from bazaar, because there are environments.");
```
Comment (Erkin97) [inline commons/src/Commons.cpp]: They are many reasons of internal server error

File: hub/forms/DlgAbout.ui [hunk 3]
```diff
@@ -53,8 +53,14 @@
        <item row="4" column="1">
         <widget class="QLabel" name="lbl_rh_version_val">
          <property name="text">
-          <string>undefined</string>
+          <string>Undefined</string>
```
Comment (Erkin97) [inline hub/forms/DlgAbout.ui]: upper letter makes ui design looks not so good, [i will attach screenshot]

File: hub/forms/DlgLogin.ui [hunk 4]
```diff
@@ -104,7 +104,7 @@
        <item row="0" column="1">
         <widget class="QCheckBox" name="cb_show_pass">
          <property name="text">
-          <string>Show password when type</string>
+          <string>Show password while typing</string>
```
Comment (Erkin97) [inline hub/forms/DlgLogin.ui]: better to do just `Show password`

File: hub/src/DownloadFileManager.cpp.autosave [hunk 5]
```diff
@@ -1,84 +0,0 @@
-#include "DownloadFileManager.h"
```
Comment (Erkin97) [inline hub/src/DownloadFileManager.cpp.autosave]: remove this file

Comment (Erkin97) [reply hub/src/DownloadFileManager.cpp.autosave]: whole file

File: hub/src/DownloadFileManager.cpp [hunk 6]
```diff
@@ -52,7 +52,7 @@ CDownloadFileManager::reply_finished() {
 
   if (m_network_reply->error() != QNetworkReply::NoError) {
     CNotificationObserver::Instance()->Error(
-          tr("Download file error. %1").arg(m_network_reply->errorString()), DlgNotification::N_NO_ACTION);
+          tr("File Download Error. %1").arg(m_network_reply->err    orString()), DlgNotification::N_NO_ACTION);
```
Comment (Erkin97) [inline hub/src/DownloadFileManager.cpp]: fix this

File: hub/src/DlgAbout.cpp [hunk 7]
```diff
@@ -386,10 +386,10 @@ void
 DlgAbout::got_chrome_version_sl(QString version) {
   ui->lbl_chrome_version_val->setText(version);
   if(version == "undefined"){
-      ui->btn_chrome->setText(tr("Install Chrome"));
+      ui->btn_chrome->setText(tr("Install Google Chrome"));
```
Comment (Erkin97) [inline hub/src/DlgAbout.cpp]: Constant name for component is "Chrome"

File: hub/forms/DlgLogin.ui (hunk 4 above)
Comment (Erkin97) [reply hub/forms/DlgLogin.ui]: please remove all changes from .ui files

**Pull request closed without merging**